selfdrive/version.py

```diff
@@ -45,7 +45,7 @@ def get_git_remote(default: Optional[str] = None) -> Optional[str]:
 prebuilt = os.path.exists(os.path.join(BASEDIR, 'prebuilt'))
 
 training_version: bytes = b"0.2.0"
-terms_version: bytes = b"2"
+terms_version: bytes = b"3"
```

Comment:
For testing? Remove so we don't accidentally merge it in.

Reply:
Well it is a new version :). Ok I'll remove it